mjit_worker.c

```diff
@@ -1138,15 +1138,16 @@ int rb_workqueue_register(unsigned flags, rb_postponed_job_func_t , void *);
 // We're lazily copying cache values from main thread because these cache values
 // could be different between ones on enqueue timing and ones on dequeue timing.
 bool
-mjit_copy_cache_from_main_thread(const rb_iseq_t *iseq, struct rb_call_cache *cc_entries, union iseq_inline_storage_entry *is_entries)
+mjit_copy_cache_from_main_thread(const rb_iseq_t *iseq, const struct rb_callcache * const *cc_entries,
+                                 union iseq_inline_storage_entry *is_entries)
 {
     mjit_copy_job_t *job = &mjit_copy_job; // just a short hand
 
     CRITICAL_SECTION_START(3, "in mjit_copy_cache_from_main_thread");
     job->finish_p = true; // disable dispatching this job in mjit_copy_job_handler while it's being modified
     CRITICAL_SECTION_FINISH(3, "in mjit_copy_cache_from_main_thread");
 
-    job->cc_entries = cc_entries;
+    job->cc_entries = (void *)cc_entries; // TODO
```

Comment:
Is this still inevitable?

Reply:
2 points.
(1) we need to copy cc_entries for GC. I'll write it in comment. f48016755a
(2) type mimatch because of `const`, so it is needed to solve.